includes/Views/DspaceBooks.php

```diff
@@ -219,120 +219,109 @@ private function displayLinks( $start_here, $search_term ) {
 	 * @return type|mixed|string
 	 */
 	private function licensePicker( $dspace_array, $authors ) {
-		$v3       = false;
-		$endpoint = 'https://api.creativecommons.org/rest/1.5/';
-		$expected = array(
-			'cc0'      => array(
-				'license'     => 'zero',
-				'commercial'  => 'y',
-				'derivatives' => 'y',
-			),
-			'by'       => array(
-				'license'     => 'standard',
-				'commercial'  => 'y',
-				'derivatives' => 'y',
-			),
-			'by-sa'    => array(
-				'license'     => 'standard',
-				'commercial'  => 'y',
-				'derivatives' => 'sa',
-			),
-			'by-nd'    => array(
-				'license'     => 'standard',
-				'commercial'  => 'y',
-				'derivatives' => 'n',
-			),
-			'by-nc'    => array(
-				'license'     => 'standard',
-				'commercial'  => 'n',
-				'derivatives' => 'y',
-			),
-			'by-nc-sa' => array(
-				'license'     => 'standard',
-				'commercial'  => 'n',
-				'derivatives' => 'sa',
-			),
-			'by-nc-nd' => array(
-				'license'     => 'standard',
-				'commercial'  => 'n',
-				'derivatives' => 'n',
-			),
-		);
-		// interpret string as an object
-		$xml = $dspace_array;
-
-		if ( $xml ) {
-			$license = $this->metadataToCsv( $xml, 'dc.rights.uri' );
-			$license = strtolower( $license );
-			$license = explode( "/", $license );
-			$license = $license[4];
-			$title   = $this->metadataToCsv( $xml, 'dc.title' );
-			$lang    = $this->metadataToCsv( $xml, 'dc.language' );;
-		}
-
-		// nothing meaningful to hit the api with, so bail
-		if ( ! array_key_exists( $license, $expected ) ) {
-			// try this first
-			try {
-				$license = $this->v3license( $license );
-				$v3      = true;
-			} catch ( \Exception $exc ) {
-				\error_log( $exc->getMessage() );
-
-				// get out of here
-				return $license;
+		$uricheck = $this->metadataToCsv( $dspace_array, 'dc.rights.uri' );
+		if ( $uricheck ) {
+			$v3       = false;
+			$endpoint = 'https://api.creativecommons.org/rest/1.5/';
+			$expected = array(
+				'cc0'      => array(
+					'license'     => 'zero',
+					'commercial'  => 'y',
+					'derivatives' => 'y',
+				),
+				'by'       => array(
+					'license'     => 'standard',
+					'commercial'  => 'y',
+					'derivatives' => 'y',
+				),
+				'by-sa'    => array(
+					'license'     => 'standard',
+					'commercial'  => 'y',
+					'derivatives' => 'sa',
+				),
+				'by-nd'    => array(
+					'license'     => 'standard',
+					'commercial'  => 'y',
+					'derivatives' => 'n',
+				),
+				'by-nc'    => array(
+					'license'     => 'standard',
+					'commercial'  => 'n',
+					'derivatives' => 'y',
+				),
+				'by-nc-sa' => array(
+					'license'     => 'standard',
+					'commercial'  => 'n',
+					'derivatives' => 'sa',
+				),
+				'by-nc-nd' => array(
+					'license'     => 'standard',
+					'commercial'  => 'n',
+					'derivatives' => 'n',
+				),
+			);
+			// interpret string as an object
+			$xml = $dspace_array;
```

Comment:
`$xml` variable is not xml. It is an array...please keep as `$dspace_array` for clarity?